test/kira_web/controllers/webhooks/gitlab_controller/merge_request_webhook_test.exs

```diff
@@ -50,4 +50,57 @@ defmodule KiraWebTest.Webhooks.GitlabController.MergeRequestWebhookTest do
       assert MergeRequestQueries.get_merge_request!(merge_request["uid"])
     end
   end
+
+  describe "update merge request webhook" do
+    setup %{project: project, author: author} do
+      {:ok,
+       merge_request: insert(:merge_request, project: project, author: author)}
+    end
+
+    test "valid merge request data is handled", %{
+      merge_request: merge_request,
+      conn: conn
+    } do
+      merge_request_payload =
+        merge_request
```

Comment:
Why are you not using something like `string_params_for(:merge_request, author: author)`?

Reply:
It's because `string_params_for` don't save information to the database. Here we test record updating, so we need the record. I use 'insert' (which saves information to the database) and converting data to needed structure.